source/ada/lsp-lal_utils.ads

```diff
@@ -162,4 +162,25 @@ package LSP.Lal_Utils is
    --  parent. It sets Imprecise_Results to True if any request returns
    --  imprecise results.
 
+   function Check_If_Is_Enum_Literal
```

Comment:
I propose to remove the prefix "Check_If_" in both this function and the name: `Is_Enum_Literal` is clear...